types/lodash-es/index.d.ts

```diff
@@ -4,304 +4,305 @@
-export { default as toNumber } from "./toNumber";
-export { default as toPairs } from "./toPairs";
-export { default as toPairsIn } from "./toPairsIn";
-export { default as toPath } from "./toPath";
-export { default as toPlainObject } from "./toPlainObject";
-export { default as toSafeInteger } from "./toSafeInteger";
-export { default as toString } from "./toString";
-export { default as toUpper } from "./toUpper";
-export { default as transform } from "./transform";
-export { default as trim } from "./trim";
-export { default as trimEnd } from "./trimEnd";
-export { default as trimStart } from "./trimStart";
-export { default as truncate } from "./truncate";
-export { default as unary } from "./unary";
-export { default as unescape } from "./unescape";
-export { default as union } from "./union";
-export { default as unionBy } from "./unionBy";
-export { default as unionWith } from "./unionWith";
-export { default as uniq } from "./uniq";
-export { default as uniqBy } from "./uniqBy";
-export { default as uniqWith } from "./uniqWith";
-export { default as uniqueId } from "./uniqueId";
-export { default as unset } from "./unset";
-export { default as unzip } from "./unzip";
-export { default as unzipWith } from "./unzipWith";
-export { default as update } from "./update";
-export { default as updateWith } from "./updateWith";
-export { default as upperCase } from "./upperCase";
-export { default as upperFirst } from "./upperFirst";
-export { default as values } from "./values";
-export { default as valuesIn } from "./valuesIn";
-export { default as without } from "./without";
-export { default as words } from "./words";
-export { default as wrap } from "./wrap";
-export { default as xor } from "./xor";
-export { default as xorBy } from "./xorBy";
-export { default as xorWith } from "./xorWith";
-export { default as zip } from "./zip";
-export { default as zipObject } from "./zipObject";
-export { default as zipObjectDeep } from "./zipObjectDeep";
-export { default as zipWith } from "./zipWith";
+export { default as add } from './add';
+export { default as after } from './after';
+export { default as ary } from './ary';
+export { default as assign } from './assign';
+export { default as assignIn } from './assignIn';
+export { default as assignInWith } from './assignInWith';
+export { default as assignWith } from './assignWith';
+export { default as at } from './at';
+export { default as attempt } from './attempt';
+export { default as before } from './before';
+export { default as bind } from './bind';
+export { default as bindAll } from './bindAll';
+export { default as bindKey } from './bindKey';
+export { default as camelCase } from './camelCase';
+export { default as capitalize } from './capitalize';
+export { default as castArray } from './castArray';
+export { default as ceil } from './ceil';
+export { default as clamp } from './clamp';
+export { default as clone } from './clone';
+export { default as chain } from './chain';
```

Comment:
These appear to be alphabetized, so `chain` and `chunk` should probably go before `clamp` and `clone`.